snetd/cmd/components.go

```diff
@@ -267,7 +276,12 @@ func (components *Components) GrpcInterceptor() grpc.StreamServerInterceptor {
 }
 
 //Metering end point authentication is now mandatory for daemon
-func (components *Components) verifyMeteringConfigurations(serviceURL string,groupId string) (ok bool, err error) {
+func (components *Components) verifyAuthenticationSetUpForFreeCall(serviceURL string,groupId string) (ok bool, err error) {
+
+	if _, err = crypto.HexToECDSA(config.GetString(config.PvtKeyForMetering)); err != nil {
+		return false, errors.New("you need a specify a valid private key 'pvt_key_for_metering' given by you as part " +
```

Comment:
Change message to "as part of service publication process". We dont need to mention free calls as this is required for any payment model